README.md

```diff
@@ -74,6 +74,9 @@ more information.
 
         $ make run
 
+5. When signing up for the app locally, you can view sent system emails at
+   http://localhost:1080/.
+
```

Comment:
Perhaps we want to use the same language as in the Docker instructions, for consistency? Something like:
```
Once the server is up and running, the app will be accessible
at `http://localhost:3000/`.  Email messages will be visible in MailCatcher
at `http://localhost:1080/`.
```